CHANGELOG.md

```diff
@@ -1,5 +1,169 @@
+* [ENHANCEMENT] Update the `memcached-exporter` to `v0.11.2`. #4570
+* [ENHANCEMENT] Autoscaling: Add `autoscaling_query_frontend_memory_target_utilization`, `autoscaling_ruler_query_frontend_memory_target_utilization`, and `autoscaling_ruler_memory_target_utilization` configuration options, for controlling the corresponding autoscaler memory thresholds. Each has a default of 1, i.e. 100%. #4612
+* [ENHANCEMENT] Distributor: add ability to set per-distributor limits via `distributor_instance_limits` using runtime configuration. #4627
+* [BUGFIX] Add missing query sharding settings for user_24M and user_32M plans. #4374
+
+### Mimirtool
+
+* [ENHANCEMENT] Backfill: mimirtool will now sleep and retry if it receives a 429 response while trying to finish an upload due to validation concurrency limits. #4598
+* [ENHANCEMENT] `gauge` panel type is supported now in `mimirtool analyze dashboard`. #4679
+* [ENHANCEMENT] Set a `User-Agent` header on requests to Mimir or Prometheus servers. #4700
+
+### Mimir Continuous Test
+
+* [FEATURE] Allow continuous testing of native histograms as well by enabling the flag `-tests.write-read-series-test.histogram-samples-enabled`. The metrics exposed by the tool will now have a new label called `type` with possible values of `float`, `histogram_float_counter`, `histogram_float_gauge`, `histogram_int_counter`, `histogram_int_gauge`, the list of metrics impacted: #4457
+  * `mimir_continuous_test_writes_total`
+  * `mimir_continuous_test_writes_failed_total`
+  * `mimir_continuous_test_queries_total`
+  * `mimir_continuous_test_queries_failed_total`
+  * `mimir_continuous_test_query_result_checks_total`
+  * `mimir_continuous_test_query_result_checks_failed_total`
+* [ENHANCEMENT] Added a new metric `mimir_continuous_test_build_info` that reports version information, similar to the existing `cortex_build_info` metric exposed by other Mimir components. #4712
+* [ENHANCEMENT] Add coherency for the selected ranges and instants of test queries. #4704
+
+### Query-tee
+
+### Documentation
+
+* [CHANGE] Clarify what deprecation means in the lifecycle of configuration parameters. #4499
+* [CHANGE] Update compactor `split-groups` and `split-and-merge-shards` recommendation on component page. #4623
+* [FEATURE] Add instructions about how to configure native histograms. #4527
+* [ENHANCEMENT] Runbook for MimirCompactorHasNotSuccessfullyRunCompaction extended to include scenario where compaction has fallen behind. #4609
+* [ENHANCEMENT] Add explanation for QPS values for reads in remote ruler mode and writes generally, to the Ruler dashboard page. #4629
+* [ENHANCEMENT] Expand zone-aware replication page to cover single physical availability zone deployments. #4631
+* [FEATURE] Add instructions to use puppet module. #4610
+
+### Tools
+
+* [ENHANCEMENT] tsdb-index: iteration over index is now faster when any equal matcher is supplied. #4515
+
 ## 2.8.0-rc.2
```

Comment:
Question to reviewers: should we remove rc changelog?

Reply:
I don't see an entry for previous RC releases, so yep, I think we should remove the 2.8 RCs.